src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerRequestTranscriptionIntTest.java

```diff
@@ -166,20 +161,6 @@ private void assertTranscriptionWorkflow(TranscriptionWorkflowEntity transcripti
         assertEquals(expectedWorkflowActor, transcriptionWorkflowToCheck.getWorkflowActor());
     }
 
-    private void assertAudit(int expected) {
```

Comment:
The deletions in here need to tested in a different way and not removed, as per the other comments.

Reply:
Code updated for tests